akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -129,12 +129,11 @@ object ActorMaterializer {
    * INTERNAL API
    */
   private[akka] def downcast(materializer: Materializer): ActorMaterializer =
-    materializer match {
+    materializer match { //FIXME this method is going to cause trouble for other Materializer implementations
```

Comment:
I don't follow what is the problem here, but whatever. Keep this FIXME here then.